diff --git a/third_party/freetype2/BUILD.gn b/third_party/freetype2/BUILD.gn
index 631b909..9251143 100644
--- a/third_party/freetype2/BUILD.gn
+++ b/third_party/freetype2/BUILD.gn
@@ -37,16 +37,8 @@ if (skia_use_system_freetype2) {
 
     defines = [ "FT2_BUILD_LIBRARY" ]
     if (target_cpu == "wasm") {
-      public_defines += [
-        "FT_CONFIG_MODULES_H=<freetype-no-type1/freetype/config/ftmodule.h>",
-        "FT_CONFIG_OPTIONS_H=<freetype-no-type1/freetype/config/ftoption.h>",
-      ]
       public_include_dirs += [ "include/freetype-no-type1" ]
     } else {
-      public_defines += [
-        "FT_CONFIG_MODULES_H=<freetype-android/freetype/config/ftmodule.h>",
-        "FT_CONFIG_OPTIONS_H=<freetype-android/freetype/config/ftoption.h>",
-      ]
       public_include_dirs += [ "include/freetype-android" ]
     }
     public_include_dirs += [ "../externals/freetype/include" ]
